assets/scss/_header.scss

```diff
@@ -0,0 +1,51 @@
+.header {
+  background-image: url("../images/header-bg.svg");
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.nav {
+  &__list {
+    display: flex;
+    align-items: center;
+    list-style: none;
+    padding: 0 1rem;
+    @include media-queries("tablet") {
+      padding: 1.2rem 3rem 1.7rem;
+    }
+  }
+
+  &__item {
+    padding: 0.2rem 1rem 2rem;
+    @include media-queries("tablet") {
+      padding: 0.2rem 3rem 2rem;
+    }
+    text-decoration: none;
```

Comment:
it's not readable, at first should be all styles for mobile, so just after `padding` ->  `text-decoration` then media-queries